docs/TLS.md

```diff
@@ -15,10 +14,22 @@ In addition, one has to pass the `tls-enabled yes` option to the redisraft modul
 --loadmodule ./redisraft.so <other options> tls-enabled yes
 ```
 
-RedisRaft is smart enough to figure out which files to use for the local cert/key/ca by querying the underlying Redis server.  However, if a user wants to use different files for the client aspects vs server aspects, one can specify RedisRaft module options as well
+## Configuring RedisRaft TLS support
 
-```asm
-tls-ca-cert
-tls-cert
-tls-key
-```
\ No newline at end of file
+RedisRaft can be configured by reusing the Redis server's TLS configuration, or by configuring the RedisRaft module directly. In general, we expect people will configure the Redis server. 
```

Comment:
Proposing to reconsider the double configuration option. What use case does it support?
I don't see a scenario where a RedisRaft redis-server will ever use the `tls-client-*` settings for anything, because it's never a cluster node / replica. That being the case, perhaps we can just assume we own these parameters and avoid the need to maintain this extra complexity?

Reply:
not 100% sure I understand what you mean.

Reply:
do you mean, that we should just not have module config options and just always read it out of redis?

Reply:
removed the module configs and simplified doc

Reply:
my impression was that there was no need to have module configs, as if we would want to distinguish the keys redisraft uses from the keys redis uses on the server side we would use tls-client-* as redis wont be using them (as no replicas/clustering)

Reply:
Oh okay that's even better. 

>removed the module configs and simplified doc

Did you forget to push the commit for this? :) I still see module configs in the PR.

Reply:
yes, that also goes with the other comment, now pushed

Reply:
Ok so, we'll only use `tls-client-*` configs? If one wants to config redisraft, they set `tls-client-*` configs. I like this, makes code simpler. 

It needs another commit as far as I can see right? With this decision, we don't even need to touch `tls-key-file` , `tls-key-file-pass`, `tls-ca-cert-file` or `tls-cert-file`. There are places still using these configs. Or did I misunderstand again? :)

Reply:
we need to set tls-key... as this is the server's files (i.e. redis-server)  we can choose to set tls-client-* if we want redisraft to us different files than the server.  if we don't, we don't have to set them.

Reply:
Why don't we just ask users to set `tls-client-*` options? If they want to configure RedisRaft, they will set `tls-client-*`. If they want to use same certs with "server mode"(incoming side), they'll copy paste them as `tls-client-*`. So, RedisRaft will use them in the client mode (outgoing). Is there a downside for this? If we can do this, we just say RedisRaft TLS config is `tls-client-*`, that's it. Wdyt?

Reply:
Just to recap:

Your suggestion: No module param configs. Use Redis' `tls-client-*` if exists, otherwise use `tls-` files in RedisRaft.
Mine: No module param configs. Use Redis' 'tls-client-*` only in RedisRaft.

Maybe mine is unnecessary, differs a bit from Redis anyway.

Reply:
the point is tls-client-* can be redundant.  we can't run redis-server without the normal tls-* as it handles incoming connections itself.  only if redisraft wants to use different files, will we need to set tls-client-*, otherwise it just be duplicating entries on the command line.